client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -156,7 +156,14 @@ define( [ 'utils/utils', 'utils/deferred', 'mvc/ui/ui-misc', 'mvc/form/form-view
                                 // queue model request
                                 self.deferred.reset();
                                 self.deferred.execute( function( process ) {
-                                    self._buildModel( process, { id : id, version : version } )
+                                    if ( options.hasOwnProperty( "workflow" ) ) {
```

Comment:
@guerler Do you object to me merging as is so you can follow up with that change - or do you want this change made first?